src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawFigure();
```

Comment:
interface - это абстракция. Класс (Figure) - это реализация. По умолчанию абстракция не должна знать о своих реализациях. В будущем этот интерфейс могут реализовывать разные классы, которые хотят иметь такое поведение. Например создадим класс `Car implements Drawable` `car.drawFigure()` ? Выглядит странно, ведь автомобиль - это не фигура. `car.draw()` - отлично.
```suggestion
    void draw();
```

Reply:
Thanks for your comment. Corrected.